app/src/main/java/com/mtlepberghenov/internship_playground/screens/main/view/MainActivity.java

```diff
@@ -0,0 +1,27 @@
+package com.mtlepberghenov.internship_playground.screens.main.view;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+
+import com.mtlepberghenov.internship_playground.R;
+import com.mtlepberghenov.internship_playground.screens.welcome.WelcomeFragment;
+
+public class MainActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        startWelcomeFragment();
+    }
+
+    private void startWelcomeFragment() {
+        if (findViewById(R.id.main_container) != null) {
```

Comment:
is there any cases when it might be null?

Reply:
Hi! It will be null when I use other activity, I mean when use other orientation: activity(land) and it don't have the container.